docs/research/feature-gap-analysis.md

```
@@ -0,0 +1,845 @@
+|----------|------------|---------|---------------------|
+| `configureUps(config)` | UPS configuration | varies | **NO** |
+
+---
+
+## C. All GraphQL Subscriptions Available
+
+Every subscription channel identified with update intervals and event triggers.
+
+### C.1 PubSub Channel Definitions (from source code)
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)